<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesSelection</class>
 <widget class="QMainWindow" name="FilesSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background: white;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="toolbarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
background:#73c259;
}</string>
        </property>
        <widget class="QLabel" name="labelWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>91</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
font-familiy: Roboto;
font-size: 20px;
font:bold;
color: white;
}</string>
         </property>
         <property name="text">
          <string>PDS Pro</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,0,0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="profileImageLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>75</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>75</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(83, 255, 218);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>Nessuna immagine profilo</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="newDocumentButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Nuovo Documento</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLabel" name="nicknameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>23</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>4</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">*{

font-familiy: Roboto;
font-size: 12px;
font:bold;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Il tuo nickname</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QPushButton" name="newFileFromLink">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Aggiungi File da Uri</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="changeProfileButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Modifica profilo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>500</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout" rowminimumheight="0" columnminimumwidth="0">
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="fileListLabel">
             <property name="styleSheet">
              <string notr="true">*{
font-familiy: Roboto;
font-size: 24px;
font:bold;
}</string>
             </property>
             <property name="text">
              <string>Lista dei tuoi file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="fileListWidget">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(231, 231, 231);</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="currentRow">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
